### DISCUSSION TITLE
AutoRespond

### DISCUSSION THREAD

**Discussion by Ryzz3nn** [ORelio/Minecraft-Console-Client#2001] [Q&A]
so what im trying to do here is to get the client to send that in chat whenever i type the match in the chat but i cant get this to work at all

[Match]
match=bajs
actionother=send Ogge Bajs!!

**Comment by ORelio** [comment]
See [detecting chat messages](https://github.com/MCCTeam/Minecraft-Console-Client/tree/master/MinecraftClient/config#detecting-chat-messages).
Once solved, use `action` instead of `actionother`.
You can also try enabling `debugmessages` in the setting file while diagnosing your setup.
